app/server.go

```diff
@@ -0,0 +1,21 @@
+package app
+
+import (
+	"github.com/gorilla/mux"
+	"log"
+	"net/http"
+	"wizeline/routes"
+)
+
+var Router *mux.Router
```

Comment:
By convention, all the Exported variables, constants, functions, etc, should have a comment with a description:
```go
// Router - description here...
var Router *mux.Router
```